IAteIt/IAteIt/View/MealDetail/MealDetailView.swift

```diff
@@ -36,7 +36,6 @@ struct MealDetailView: View {
                     TabView {
                         ForEach(meal.plates, id: \.id) { plate in
                             PhotoCardView(plate: plate)
-                                .padding(.horizontal, .paddingHorizontal)
                                 .contextMenu {
```

Comment:
아래와 같은 코드를 추가하여 contextMenuPreview의 cornerRadius를 사진과 동일하게 주었더니 해결되는 것 같습니다!
```suggestion
                                .contentShape(.contextMenuPreview, RoundedRectangle(cornerRadius: 20))
                                .contextMenu {
```

Reply:
와 천재신가요... 배워갑니다,,,

Reply:
Constant에 추가해서 올리겠슴다🫡